### PR TITLE
Send AMP events to main Keen instance

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

File: sinks/keen-redux.js [hunk 1]
```diff
@@ -61,7 +86,8 @@ export default (message) => {
 				return resolve();
 			}
 
-			const eventName = getEventName(category, action);
+			const product = message.ingest._body.context.product.toLowerCase();
```
Comment (commuterjoy) [inline sinks/keen-redux.js]: product should be there, but might not be - i'd just to a check for it

Comment (commuterjoy) [inline sinks/keen-redux.js]: ok, thanks - ship away

**Comment by adambraimbridge** [comment]
👍

**Comment by adambraimbridge** [comment]
Good catch 👍

**Pull request merged**